k8s/yaml/servers.yaml

```diff
@@ -1,15 +1,20 @@
+---
+# Source: seldon-core-v2-servers/templates/seldon-v2-servers.yaml
 apiVersion: mlops.seldon.io/v1alpha1
 kind: Server
 metadata:
   name: mlserver
+  namespace: 'default'
```

Comment:
:wrench: We probably want this to be templated nicely... Will need to look into why it isn't being